spec/features/proposal/collection_draft_proposals/collection_draft_proposal_update_spec.rb

```diff
@@ -1,4 +1,4 @@
-describe 'Collection Draft Proposal Update', js: true do
+describe 'Collection Draft Proposal Update', reset_provider: true, js: true do
```

Comment:
I mentioned this in of Ryan's pull requests, does adding/keeping js:true slow down the tests in your opinion?   I thought I noticed a significant difference one day when I was testing.

Reply:
Yes it does. I think we should remove it where possible, but if your integration test requires things that the js is doing, then it is necessary. I have also found it necessary for capybara screenshots, but that can be used temporarily.

Reply:
this test _might not_ need `js: true` but I'd rather not change it at the moment. with one of the future draft mmt tickets, I'll review if all the tests for proposals need it or not.